README.org

```diff
@@ -18,3 +18,6 @@ Here's a table describing the bindings
 | =ediff-suspend=             | =z=              | =C-z=          |
 | scroll left                 | =>=              | =zh=           |
 | scroll right                | =<=              | =zl=           |
+| copy A region to B's region | a                | l              |
+| copy B region to A's region | b                | h              |
```

Comment:
i didn't really consider this, most common is 2 file  i think
and simple home row single handed operation would be handy,
maybe I should leave the original a b c as well